### PR TITLE
fix changelog

### PR DESCRIPTION
**Pull request by cortisiko**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions,_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,27 +2,24 @@
...
-- [#5613](https://github.com/MetaMask/metamask-mobile/pull/5613): [FIX] Analytics event interface
+- [#5605](https://github.com/MetaMask/metamask-mobile/pull/5605): [FIX] Handle ENS Address Error
 - [#5600](https://github.com/MetaMask/metamask-mobile/pull/5600): [IMPROVEMENT] Refactor ProtectYourWallet section in the security section
-- [#5586](https://github.com/MetaMask/metamask-mobile/pull/5586): Stablize Addressbook and Networks flow tests
-- [#5594](https://github.com/MetaMask/metamask-mobile/pull/5594): bumped ua-parser-js to 0.7.33
-- [#5580](https://github.com/MetaMask/metamask-mobile/pull/5580): Bump cookiejar from 2.1.2 to 2.1.4
-- [#5471](https://github.com/MetaMask/metamask-mobile/pull/5471): Bump luxon from 3.1.1 to 3.2.1
-- [#5450](https://github.com/MetaMask/metamask-mobile/pull/5450): Bump json5 from 1.0.1 to 1.0.2
+- [#5586](https://github.com/MetaMask/metamask-mobile/pull/5586): [IMPROVEMENT] Stablize Addressbook and Networks flow tests
+- [#5594](https://github.com/MetaMask/metamask-mobile/pull/5594): [IMPROVEMENT] bumped ua-parser-js to 0.7.33
```
Comment (Cal-L) [inline CHANGELOG.md:19]: Let's change these version bumps to `[UPGRADE]`

**Review by Cal-L — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (cortisiko) [reply CHANGELOG.md]: done!

**Review by sethkfman — COMMENTED** [review]
LGTM

**Pull request merged**